src/cron/cron.ts

```diff
@@ -153,8 +152,10 @@ async function daily(bot: Bot<BotContext>) {
 }
 
 async function reminder(bot: Bot<BotContext>) {
-  await schedule.gracefulShutdown();
```

Comment:
what about this one

Reply:
we don’t use node-schedule anymore

Reply:
I mean it's used to delete all memorized crones right so do we should use smth like this or deno cron itself forgets

Reply:
it’ll be cleaned by gc

Reply:
sorry but what is "gc"

Reply:
`croner` doesn’t have such a functionality

Reply:
garbage collector